README.md

```diff
@@ -1,35 +1,61 @@
-| OpenVDB               | Not Started | https://github.com/AcademySoftwareFoundation/openvdb     |                                                                     |                    | https://crates.io/crates/openvdb             |
-| OpenTimelineIO        | In Progress | https://github.com/PixarAnimationStudios/OpenTimelineIO  | https://github.com/PixarAnimationStudios/OpenTimelineIO/tree/c-otio |                    | https://crates.io/crates/opentimelineio      |
-| Open Shading Language | Not Started | https://github.com/imageworks/OpenShadingLanguage        |                                                                     |                    | https://crates.io/crates/openshadinglanguage |
-| OpenImageIO           | In Progress | https://github.com/OpenImageIO/oiio                      | https://github.com/OpenImageIO/oiio/pull/2748                       |                    | https://crates.io/crates/openimageio         |
-| OpenSubdiv            | Not Started | https://github.com/PixarAnimationStudios/OpenSubdiv      |                                                                     |                    | https://crates.io/crates/opensubdiv          |
-| Ptex                  | Not Started | https://github.com/wdas/ptex                             |                                                                     |                    | https://crates.io/crates/ptex                |
-| SeExpr                | Not Started | https://github.com/wdas/SeExpr                           |                                                                     |                    | https://crates.io/crates/seexpr              |
-| USD                   | In Progress | https://github.com/PixarAnimationStudios/USD             |                                                                     |  https://github.com/luke-titley/usd-rs | https://crates.io/crates/usd                 |
 
-Contributions
-=============
 
-All contributions are welcome. However, the projects that we wrap require a contributor license agreement. A contributor to our repositories agrees to contributing under the license of the repository and is legally able to contribute.
+# Rust Bindings Working Group
+
+The Rust bindings working group is dedicated to create C and Rust bindings for C++ libraries used by the media and entertainment industry.
+
+Goals of the WG are:
+
+- Create a framework for collaborating with projects and working groups to create consistent C and Rust bindings, and to facilitate creating the bindings.
+- Create tooling to help build the C and Rust bindings.
+- Create a set of conventions and rules to create a consistant and ergonomic C and Rust interface for the wrapped libraries.
+- To have the Rust bindings easily accessable through Rust's crates.io repository, and to give ownership of those bindings to the ASWF.
+
+Non-goals of the WG are:
+
+- Converting or reimplementing current C++ libraries into Rust libraries.
```

Comment:
```suggestion
- Reimplementing all awsf C++ libraries into Rust.
```
I suggest changing this to be more vague, because we're still not 100% sure about imath. imath could be an exception...